eng/testing/debug-dump-template.md

```diff
@@ -1,127 +1,145 @@
-# Debugging a CI dump
+# Get the dump
 
-This document describes how to debug a CI/PR test dump by downloading assets from helix, using a dotnet tool called `runfo`.
+Click the link to the dump on the `Helix Test Logs` tab in Azure DevOps. This is the same place you got these instructions from.
 
-## What is runfo?
+# Get the Helix payload
 
-Runfo is a dotnet global tool that helps get information about helix test runs and azure devops builds. For more information see [this](https://github.com/jaredpar/runfo/tree/master/runfo#runfo)
+[Runfo](https://github.com/jaredpar/runfo/tree/master/runfo#runfo) helps get information about helix test runs and azure devops builds. We will use it to download the payload and symbols:
+```script
```

Comment:
nit: only this file in the repo is using ` ```script `, which is not a [known language alias](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml). In other documents, ` ```sh ` and ` ```cmd ` are used.

Reply:
According to that file, `cmd` isn't either! I'll use `sh` which will at least be right for Unix. it doesn't really matter.

Reply:
cmd definitely is.
https://github.com/github/linguist/blob/8fc4898d34dbe9323bc1bb7971652998d3d39b4f/lib/linguist/languages.yml#L449

```cmd
:: this is cmd

setx FOO=BAR
echo %FOO%
```

Reply:
Extensions are also included in language name, so cmd is valid https://github.com/dotnet/runtime/pull/46493#discussion_r550953706.